src/test/java/uk/gov/hmcts/sptribs/controllers/CaseManagementControllerTest.java

```diff
@@ -45,107 +48,111 @@ class CaseManagementControllerTest {
+        final DssCaseData dssCaseData = mapper.readValue(caseDataJson, DssCaseData.class);
+        final CaseData caseData = CaseData.builder().dssCaseData(dssCaseData).build();
+        final Map<String, Object> caseDataMap = new ConcurrentHashMap<>();
+        final CaseResponse caseResponse = CaseResponse.builder().caseData(caseDataMap).build();
 
         caseDataMap.put(CASE_DATA_CIC_ID, caseData);
-        CaseResponse caseResponse = CaseResponse.builder().caseData(caseDataMap).build();
         caseResponse.setId(TEST_CASE_ID);
         caseResponse.setStatus(null);
 
         when(caseManagementService.updateCase(eq(CASE_TEST_AUTHORIZATION), eq(Event.UPDATE),
             any(CaseData.class), eq(TEST_CASE_ID))).thenReturn(caseResponse);
 
-        ResponseEntity<?> preUpdateCaseResponse = caseManagementController.updateCase(
+        final ResponseEntity<?> preUpdateCaseResponse = caseManagementController.updateCase(
             TEST_CASE_ID,
             CASE_TEST_AUTHORIZATION,
             Event.UPDATE,
             caseData.getDssCaseData()
         );
 
+        final CaseResponse testPreUpdResponse = (CaseResponse) preUpdateCaseResponse.getBody();
+        assertEquals(TEST_CASE_EMAIL_ADDRESS, dssCaseData.getSubjectEmailAddress());
 
-        CaseResponse testPreUpdResponse = (CaseResponse) preUpdateCaseResponse.getBody();
-        //assertEquals(TEST_CASE_EMAIL_ADDRESS, caseData.getDssCaseData().getSubjectEmailAddress());
-
-        CaseData caseDataUpdate = (CaseData) testPreUpdResponse.getCaseData().get(CASE_DATA_CIC_ID);
+        assertThat(testPreUpdResponse).isNotNull();
+        final CaseData caseDataUpdate = (CaseData) testPreUpdResponse.getCaseData().get(CASE_DATA_CIC_ID);
         caseDataUpdate.getDssCaseData().setSubjectEmailAddress(TEST_UPDATE_CASE_EMAIL_ADDRESS);
 
-        ResponseEntity<?> postUpdateCaseResponse = caseManagementController.updateCase(
+        final ResponseEntity<?> postUpdateCaseResponse = caseManagementController.updateCase(
             TEST_CASE_ID,
             CASE_TEST_AUTHORIZATION,
             Event.UPDATE,
             caseDataUpdate.getDssCaseData()
         );
 
-        CaseResponse caseDataUpdateResponse = (CaseResponse) (postUpdateCaseResponse.getBody());
+        final CaseResponse caseDataUpdateResponse = (CaseResponse) (postUpdateCaseResponse.getBody());
+        assertThat(caseDataUpdateResponse).isNotNull();
 
-        CaseData caseDataUpdatedFromResponse = (CaseData) (caseDataUpdateResponse.getCaseData().get(CASE_DATA_CIC_ID));
+        final CaseData caseDataUpdatedFromResponse = (CaseData) (caseDataUpdateResponse.getCaseData().get(CASE_DATA_CIC_ID));
 
         assertEquals(
             caseDataUpdatedFromResponse.getDssCaseData().getSubjectEmailAddress(),
             caseDataUpdate.getDssCaseData().getSubjectEmailAddress()
         );
         assertEquals(TEST_UPDATE_CASE_EMAIL_ADDRESS, caseDataUpdate.getDssCaseData().getSubjectEmailAddress());
```

Comment:
I don't think this assert is necessary since we know that we had ran caseDataUpdate.getDssCaseData().setSubjectEmailAddress(TEST_UPDATE_CASE_EMAIL_ADDRESS) earlier in the test.